media-libs/libffado/libffado-2.4.0.ebuild

```diff
@@ -0,0 +1,118 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6} )
+
+inherit eutils python-single-r1 scons-utils toolchain-funcs udev multilib-minimal
+
+DESCRIPTION="Driver for IEEE1394 (Firewire) audio interfaces"
+HOMEPAGE="http://www.ffado.org"
+
+if [ "${PV}" = "9999" ]; then
+	inherit subversion
+	ESVN_REPO_URI="http://subversion.ffado.org/ffado/trunk/${PN}"
+else
+	SRC_URI="http://www.ffado.org/files/${P}.tgz"
+	KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~x86"
```

Comment:
compared to 2.3.0 you're dropping a few keywords here, is that caused by missing keyworded deps or an oversight?

Reply:
I dropped the keywords missing from dev-python/PyQt5 (which were present in PyQt4, which 2.3.0 depended on). Although technically this version will work with qt4 too and we could allow both deps, qt4 is going away so I figured it makes more sense to take the plunge to qt5-only.